### ISSUE TITLE
NaN0...log name after repeated start of script.

### ISSUE THREAD

**Issue by ddorkin-issart** [iccicci/rotating-file-stream#2]
Example of file with current module:

	var fs = require('fs');
	var i = 0;
	var logFileName = 'app.log';
	var handler = function(err) {
	    if (err) {
	        console.log(err.stack());
	        process.abort();
	    }
	};
	
	setInterval(function() {
	    var data = ++i;
	    fs.appendFile(logFileName, data + '\n', handler);
	});
	
	var rfs = require('rotating-file-stream');
	var stream = rfs(logFileName, {
	    size: '1K',
	    interval: '1d'
	});

When I start it in first time:

	sudo node app.js

I receive one file - app.log. On second start (if file app.js is bigger than 1 kilobyte already after first start) I receive another file:
        NaN0NaN0NaN-0NaN0NaN-01-app.log

It seems like incorrect date parsing or something else.

**Comment by iccicci** [comment]
Thank you,

I'll fix this asap.
I'going to start working on this in next five hours.

**Comment by ddorkin-issart** [comment]
Thanks for quick answer.

**Comment by iccicci** [comment]
The bug was fixed; I'll release v1.0.4 in a while.

Anyway you have not to use fs.appendFile: you have to use stream.write on the stream rotating-file-stream returns for you.

I think your code should look like follows

```
var fs = require('fs');
var i = 0;
var logFileName = 'app.log';
var handler = function(err) {
    if (err) {
        console.log(err.stack());
        process.abort();
    }
};

var rfs = require('./');
var stream = rfs(logFileName, {
    size: '1K',
    interval: '1d'
});

setInterval(function() {
    var data = ++i;
    stream.write(data + '\n', handler);
});
```

**Comment by iccicci** [comment]
I think the bug was fixed.

Due missing of feedback, I close
